.gitignore

```diff
@@ -31,6 +31,7 @@ data/Models/Adult/*.zobj
 data/Models/Adult/Pieces
 data/Models/Child/*.zobj
 data/Models/Child/Pieces
+data/Music/*
```

Comment:
There's a separate .gitignore in data/Music for excluding .seq and .meta files. To keep the readme in the Music folder, .ootrs files should be added to that file instead of the entire folder here.

Reply:
Hmm then I gotta figure out why any time I add custom music, git sees it as untracked... Anyway I'll do as you suggest. Thanks!